yarn-project/types/src/interfaces/state_provider.ts

```diff
@@ -1,8 +1,17 @@
-import { CONTRACT_TREE_HEIGHT, Fr, L1_TO_L2_MSG_TREE_HEIGHT, NOTE_HASH_TREE_HEIGHT } from '@aztec/circuits.js';
+import {
+  CONTRACT_TREE_HEIGHT,
+  Fr,
+  HISTORIC_BLOCKS_TREE_HEIGHT,
+  L1_TO_L2_MSG_TREE_HEIGHT,
+  NOTE_HASH_TREE_HEIGHT,
+  PUBLIC_DATA_TREE_HEIGHT,
+} from '@aztec/circuits.js';
 
 import { L1ToL2MessageAndIndex } from '../l1_to_l2_message.js';
+import { L2Block } from '../l2_block.js';
 import { MerkleTreeId } from '../merkle_tree_id.js';
 import { SiblingPath } from '../sibling_path.js';
+import { LowNullifierMembershipWitness } from './low_nullifier_witness.js';
 
 /**
  * Interface providing methods for retrieving information about content of the state trees.
```

Comment:
Think I made this one way back. But it pains me that the interface is `StateInfoProvider` but the file is `state_provider`...

Reply:
Created an [issue](https://github.com/AztecProtocol/aztec-packages/issues/3445). I will address this in a separate PR in which I'll include a few of these small issues.